orderer/common/multichannel/registrar.go

```diff
@@ -436,27 +435,27 @@ func (r *Registrar) newLedgerResources(configTx *cb.Envelope) (*ledgerResources,
 
 	chdr, err := protoutil.UnmarshalChannelHeader(payload.Header.ChannelHeader)
 	if err != nil {
-		return nil, errors.Wrapf(err, "error unmarshaling channel header")
+		return nil, errors.WithMessagef(err, "error unmarshaling channel header")
```

Comment:
Super nit (aka it doesn't matter at all functionally): there isn't a formatting directive (e.g. %s) so this can just be WithMessage

```suggestion
		return nil, errors.WithMessage(err, "error unmarshaling channel header")
```